LICENSE

```diff
@@ -1,20 +1,339 @@
-The MIT License (MIT)
-
-Copyright (c) 2017 Jeff Geerling
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of
-this software and associated documentation files (the "Software"), to deal in
-the Software without restriction, including without limitation the rights to
-use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
-the Software, and to permit persons to whom the Software is furnished to do so,
-subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
-FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
-COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
-IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
-CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 2, June 1991
```

Comment:
Just an heads up, if I remember correctly in order to change license you'll need the agreement from all of previous contributors who submitted their code under the MIT license.

Reply:
> you'll need the agreement from all of previous contributors who submitted their code under the MIT license.

I wonder too.
Although MIT is the most open license, not restricting any use, no need to ask for any permission, nor implying giving back, it states "copyright notice and this permission notice shall be included in all copies or substantial portions of the Software".

So, changing licenses is *not* easy, although I guess @geerlingguy wrote most of it. 
Which entitles to changing license for (at least) his code.